packages/core/src/accessanalyzer/terraformConfig/default.yaml

```diff
@@ -0,0 +1,121 @@
+# generate fake ARN
+# default can be specified using the following format:
+# {<key>?<default>}
+arnServiceMap:
```

Comment:
Q: How do we plan to keep this updated with the file in open source repo. We should define a process to fetch it from the repo or some central store for a given version, as this file gets updated very often

Reply:
please create a ticket to handle this process more gracefully to reduce the ops effort.